specs/interop/predeploys.md

```diff
@@ -94,19 +94,21 @@ The `ExecutingMessage` event represents an executing message. It MUST be emitted
 to `executeMessage`.
 
 ```solidity
-event ExecutingMessage(bytes,bytes);
+event ExecutingMessage(bytes,bytes32);
 ```
```

Comment:
Thoughts on making this `event ExecutingMessage(bytes32,bytes)` to have the fixed size in the front? We could also make it `indexed` if we want but then would need to modify https://github.com/ethereum-optimism/specs/blob/main/specs/interop/sequencer.md#direct-dependency-confirmation more to handle that, ie pull the hash from the topics[1] rather than decoding the log.data for it

Reply:
The order change is much cleaner and makes the decoding way more intuitive in my opinion. I have not added the `indexed` keyword yet as the decoding seems straightforward, but I can add it if needed and change both the solidity pseudocode and python's.

Should we also change `logHash` to `msgHash` in the solidity pseudocode? We are using `msg` in all other places instead of `log`.

Reply:
My personal style is shorter names like just `hash` but not super opinionated, i think @mds1 would like more descriptive names, if `msg` is used everywhere then `msgHash` could make sense

Reply:
Yea my ideal is short + descriptive and I think `logHash` and `msgHash` both fit that with only an extra 3 chars over `hash`. Also not super opinionated here so happy to defer :)

Reply:
I ended up calling it `msgHash` in https://github.com/ethereum-optimism/optimism/pull/11320